int16 taskId
float32 taskCost
int16 robotId
